<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="LivePlot" name="plot1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="LivePlot" name="plot2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>416</width>
      <height>221</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_simulation">
     <attribute name="title">
      <string>Simulation</string>
     </attribute>
     <widget class="QFrame" name="simFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QRadioButton" name="contSimRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Continous simulation</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="stepSimRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>41</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Step Simulation</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="stepFrame">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>191</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of steps</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="stepNoEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="samplingFrame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>391</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Sampling speed(ms):</string>
       </property>
      </widget>
      <widget class="QSlider" name="samplingSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>371</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="samplingLabel">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_source">
     <attribute name="title">
      <string>Source</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_sources">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_source">
      <property name="geometry">
       <rect>
        <x>204</x>
        <y>6</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Step</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Impuls</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sinus</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Square</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Triangle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Noise</string>
       </property>
      </item>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>209</x>
        <y>40</y>
        <width>191</width>
        <height>101</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_amplitude1">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>62</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>11</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Amplitude:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_amplitude2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>62</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_frequency2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>62</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>11</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Amplitude:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>42</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Frequency:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_amplitude3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>62</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_frequency3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>62</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_dutycycle3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>70</y>
          <width>62</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>12</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Amplitude:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>41</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Frequency:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>72</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Dutycycle:</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_addSource">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_removeSource">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>150</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_object">
     <attribute name="title">
      <string>Object</string>
     </attribute>
     <widget class="QPushButton" name="getConfigBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Config</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxObject">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Chose object:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_controller">
     <attribute name="title">
      <string>Controller</string>
     </attribute>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>P</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>12</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>201</width>
        <height>141</height>
       </rect>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_P_P">
        <property name="geometry">
         <rect>
          <x>101</x>
          <y>10</y>
          <width>91</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>16</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>P:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_feedback">
     <attribute name="title">
      <string>Feedback</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox_feedback">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>381</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable Feed Back</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="simBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Simulation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Simulation</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuConfig"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LivePlot</class>
   <extends>QWidget</extends>
   <header>gui/liveplot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
